kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/InstallKT.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+import static org.assertj.core.api.Assumptions.assumeThatCode;
+
+class InstallKT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(InstallKT.class);
+
+    public static final String SYS_PROP_IMAGE_ARCHIVE = "io.kroxylicious.operator.image.archive";
+    public static final String SYS_PROP_IMAGE_NAME = "io.kroxylicious.operator.image.name";
+    // These are normally set automatically by mvn
+    private static final String IMAGE_ARCHIVE = System.getProperty(SYS_PROP_IMAGE_ARCHIVE);
+    private static final String IMAGE_NAME = System.getProperty(SYS_PROP_IMAGE_NAME);
+    private static boolean loaded = false;
+
+    @BeforeAll
+    static void beforeAll() throws IOException, InterruptedException {
+        Assertions.setDescriptionConsumer(desc -> {
+            LOGGER.info("{}", desc);
+        });
+        assumeThat(IMAGE_ARCHIVE)
```

Comment:
It annoys me that when one of these assumptions fails the description doesn't get output in why the test got skipped. But that's seems to be JUnit's problem, not mine.

Reply:
Where are you expecting the output? Running a sample test in IntelliJ

If I skip with an assumption I see
```
org.opentest4j.TestAbortedException: assumption was not met due to: [Assumption failed] 
Expecting blank but was: "AAAAAHM+5SM"
``` 
If I used `@Disabled("disabled with annotation")`  I see
```
disabled with annotation 
```
and if I used ` @DisabledIf(value = "runTest", disabledReason = "disabled if")`  I see
```
disabled if
```

And all are marked as skipped.

Reply:
In intellij I see this:

![image](https://github.com/user-attachments/assets/e723df06-b987-4cc3-98b2-f8a4d492ca93)

I don't get anything like your

```
org.opentest4j.TestAbortedException: assumption was not met due to: [Assumption failed] 
Expecting blank but was: "AAAAAHM+5SM"
```

So I know it was skipped, but it's not clear why.

Reply:
Select one node deeper in the tree

Reply:
:cricket: :cricket: 

![image](https://github.com/user-attachments/assets/32377aa0-2100-4484-b238-9aad26341803)

Reply:
I'll experiment with the KT

Reply:
Works for me
![Screenshot 2025-05-13 at 10 09 15 am](https://github.com/user-attachments/assets/35a18850-f0ba-4006-b623-dca44a5ed17d)

Reply:
I do wonder if using `@EnabledIf` https://junit.org/junit5/docs/5.11.1/api/org.junit.jupiter.api/org/junit/jupiter/api/condition/EnabledIf.html might be a better option than `Assume` for system properties but I think its sugar at this point.